.github/workflows/cluster_endtoend_upgrade.yml

```diff
@@ -48,7 +48,7 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v2
       with:
-        go-version: 1.17.11
+        go-version: 1.18.3
```

Comment:
However, the test also failed because we used `-gateway_implementation` in the end-to-end tests. I changed how we use this flag in 4c62900640. If the major version of vtgate is below 14, we use it. Otherwise, we don't.